packages/components/src/docs/General/10-FAQ.mdx

```diff
@@ -0,0 +1,71 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="Docs/General/FAQ" />
+
+## General Questions
+
+### What is the Solid Design System?
+The Solid Design System by Union Investment establishes a robust framework for their digital presence, offering a comprehensive array of design elements and guidelines. Including navigation items, buttons, and more, it serves as an invaluable tool for designers and developers working on Union Investment’s digital offerings. Providing atomic-level components, it forms the foundational elements for ensuring uniform and seamless user interfaces across their digital products. Unlike full designs, these components enable the creation of customized interfaces while upholding visual consistency and usability standards.
+
+### What do I get from the Solid Design System
+In accordance with the principles of the Atomic Design Methodology, we furnish atoms and molecules for constructing organisms, templates, and ultimately, pages. Each element in our Solid Design System adheres to WCAG 2.2 criteria, prioritizing accessibility, and we strive to maintain their lightweight nature.
+
+### How should I be using the Solid Design System and why?
+Considering the query above, our components, styles, samples, and patterns are designed to be applicable across various web contexts, ensuring the creation of fully accessible, design standards-compliant, and lightweight websites or applications.
+
+### What are the 3 core principles of the Solid Design System?
+Solid Components, as part of the Solid Design System, serving as a robust foundation for a digital presence, built upon three core principles:
+1. 🌟 Usable & consistent
+2. 🧩 Reusable & flexible
+3. 🛠️ Maintainable & reliable
+Visit our [introduction page](https://solid-design-system.fe.union-investment.de/x.x.x/storybook/index.html?path=/docs/docs-general-introduction--docs#principles) for more details.
+
+### How do components, styles, samples, and patterns differ in our library?
+In our library, we offer four distinct types of elements:
+1. Components: These are lightweight [web components](https://developer.mozilla.org/en-US/docs/Web/API/Web_Components) constructed with lit-element.
```

Comment:
With lit, not lit-element. Link here would be good.

Reply:
done